mappings/net/minecraft/client/gui/screen/recipebook/RecipeBookResults.mapping

```diff
@@ -37,10 +37,17 @@ CLASS net/minecraft/class_513 net/minecraft/client/gui/screen/recipebook/RecipeB
 	METHOD method_2633 getRecipeBook ()Lnet/minecraft/class_3439;
 	METHOD method_2634 draw (Lnet/minecraft/class_4587;IIIIF)V
 		ARG 1 matrices
+		ARG 2 x
+		ARG 3 y
+		ARG 4 mouseX
+		ARG 5 mouseY
+		ARG 6 delta
```

Comment:
```suggestion
		ARG 6 tickDelta
```
For consistency

Reply:
hmm, i recall these all all just delta in screens

Reply:
Hmm, so it seems. Possible future PR then (tickDelta is used everywhere else).

Reply:
Moved to an issue #2669.